CONTRIBUTING.md

```diff
@@ -25,6 +25,13 @@ Avoid exception handling. Because Hyrise is not a product, we do not have to rec
 - Use forward declarations whenever possible to reduce compile time
 - We mostly use structs for PODS (plain old data structures). If it has methods, chances are that it is a class.
 
+## Using smart pointers and references
+- If an object is stored in a smart pointer, pass that pointer around as a const reference, e.g., `const std::shared_ptr<Table> &`
```

Comment:
On a different note: "It should not be necessary for a method to accept a smart pointer by copy."

Reply:
added